docs/aerodromes/Amberley.md

```diff
@@ -4,7 +4,78 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley ADC    | Amberley Tower  | 118.300         | AM_TWR        |
+| Amberley SMC    | Amberley Ground  | 129.350      | AM_GND        |
+| Amberley ACD    | Amberley Delivery  | 134.600         | AM_DEL       |
+| Amberley ATIS    |   | 123.300         | YAMB_ATIS       |
+
+
+
+## Airspace
+
+AM ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+
+## YAMB Aerodrome
+
+### SID Selection
+Aircraft planned via **BN**, **JEDDA**, **MESED**, **BOBOP**, **TATEN**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested which are often in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
+
+Aircraft will make a visual departure in the circuit direction and fly overhead YAMB to begin tracking for their initial waypoint.
+
+!!! example
+    WOLF03 was assiged the BYRON7 departure with their initial clearance. 
+    **AM ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
+    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
+
+| Departure | Initial Constraint | Route |
+| --------- | ----------| --------- |
+| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
+
+Typically pilots will request a specific departure when obtaining their airways clearance.
+
```

Comment:
Add a note to insert the name of the coded clearance in to the aircraft's Global Ops Info Field